twoliter/embedded/partyplanner

```diff
@@ -247,27 +234,51 @@ set_partition_sizes() {
   pp_size["BIOS"]="${BIOS_MIB}"
   ((offset += BIOS_MIB))
 
-  for bank in A B; do
-    pp_offset["EFI-${bank}"]="${offset}"
-    pp_size["EFI-${bank}"]="${EFI_MIB}"
-    ((offset += EFI_MIB))
+  if [[ "${in_place_updates}" == "yes" ]]; then
+    # Allocate two partition banks to support in-place updates.
+    for bank in A B; do
+      pp_offset["EFI-${bank}"]="${offset}"
+      pp_size["EFI-${bank}"]="${EFI_MIB}"
+      ((offset += EFI_MIB))
 
-    pp_offset["BOOT-${bank}"]="${offset}"
-    pp_size["BOOT-${bank}"]="${boot_mib}"
-    ((offset += boot_mib))
+      pp_offset["BOOT-${bank}"]="${offset}"
```

Comment:
This spacing seem suspicious, are they spaces or tabs?

Reply:
> This spacing seem suspicious, are they spaces or tabs?

There are no tabs in this file. It's just following the two space indentation convention of the rest of the file.